waspc/headless-test/README.md

```diff
@@ -0,0 +1,26 @@
+## Headless Test
+
+We are testing if a simple scenario works for the user each time:
+- signup
+- login
+- create a new task
+
+Check out the test file: `tests/simple.spec.ts`
+
+### How to run
+
+Running headless tests:
+```
+$ cd headless-test
+$ npm install
+$ npx playwright test
+```
+
+If something breaks, maybe the example app won't run. Try running it and see if there are any errors:
```

Comment:
I don't understand this sentence. "If something breaks" -> what? Like in setting up playwright, or smth else, what could it be? "the example app won't run" -> you mean because the test won't run? Ok, sure, if test broke before that, that makes sense.
"Try running it and see if there are any errors" -> why would I do that, what will I see? What kind of errors?